sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala

```diff
@@ -84,8 +84,15 @@ case class SetCommand(
       logWarning(
         s"Property ${SQLConf.Deprecated.MAPRED_REDUCE_TASKS} is deprecated, " +
           s"automatically converted to ${SQLConf.SHUFFLE_PARTITIONS} instead.")
-      sqlContext.setConf(SQLConf.SHUFFLE_PARTITIONS, value)
-      Seq(Row(s"${SQLConf.SHUFFLE_PARTITIONS}=$value"))
+      if (value == "-1") {
+        logWarning(
+          s"Set this property to -1 for automatically determining the number of reducers " +
+            s"is not supported, showing current ${SQLConf.SHUFFLE_PARTITIONS} instead.")
```

Comment:
actually can I suggest a small rewording a think is a little clearer?

`Setting this property to -1 for automatically determing the number of reducers is not supported.  Ignoring request, and showing  ${SQLConf.SHUFFLE_PARTITIONS} instead`